### ISSUE TITLE
Error:  'basePath' should be an absolute path.

### ISSUE THREAD

**Issue by JohnApache** [prettier/prettier-eslint-cli#208]
Command: 
```bash
$ prettier-eslint 'src/**/*.{ts,tsx}' --write 

```
Executing this command throws an exception! 

but when I use the full path, Like this:   
```bash
$ prettier-eslint '/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/src/**/*.{ts,tsx}' --write 
 
```
no error is reported, but  I want to support relative paths

When I use version **6.6.0** of **eslint**，also no error is reported;

Now I have to reduce the eslint version to solve this problem

Version: 
- `prettier-eslint-cli` version:4.7.1
- `prettier` version: 1.19.1
- `eslint` version:6.7.1

Target: 
```typescript
// src/index.ts
const a = () => {
console.log(1);
};

```

What you did: 
```bash
$ prettier-eslint 'src/**/*.{ts,tsx}' --write
```

What happened: 
Error Log:
```bash
prettier-eslint [ERROR]: There was trouble creating the ESLint CLIEngine.
prettier-eslint-cli [ERROR]: There was an error formatting "src/index.ts": 
    AssertionError [ERR_ASSERTION]: 'basePath' should be an absolute path.
        at new IgnorePattern (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/config-array/ignore-pattern.js:178:9)
        at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/config-array-factory.js:651:49)
        at _normalizeObjectConfigDataBody.next (<anonymous>)
        at ConfigArrayFactory._normalizeObjectConfigData (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/config-array-factory.js:594:20)
        at _normalizeObjectConfigData.next (<anonymous>)
        at createConfigArray (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/config-array-factory.js:338:25)
        at ConfigArrayFactory.create (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/config-array-factory.js:393:16)
        at createBaseConfigArray (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js:96:48)
        at new CascadingConfigArrayFactory (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js:211:30)
        at new CLIEngine (/Users/cuijianwei/WebstormProjects/testproject/hasaki-cli-demo/gen-demo/node_modules/eslint/lib/cli-engine/cli-engine.js:563:36)
failure formatting 1 file with prettier-eslint
error Command failed with exit code 1.
```

**Comment by JonDum** [comment]
I'm getting the same issue, it's a stopgap solution, but adding `$PWD` in front of your glob seems to fix it. might need to use `cross-env` if you support non-nix systems.
Example:
```
    "format": "prettier-eslint --write $PWD/'./src/**/*.[jt]s?(x)' && eslint --fix ."
```

**Comment by JohnApache** [comment]
@JonDum 
Thank u!
But `$PWD` does not work correctly on windows,  my current solution is to reduce the `eslint` version to @6.6.0， this works.
I don't know why the absolute path is limited in `eslint@6.7.1`?

**Comment by MrCheater** [comment]
+1

**Comment by NoPhaseNoKill** [comment]
Any updates to this? Facing same issue currently

**Comment by gsonly** [comment]
> Any updates to this? Facing same issue currently

@JonDum solution works for me, I've eslint 6.8.0

**Comment by pittersnider** [comment]
Still having the issue.

**Comment by dpwilhelmsen** [comment]
I opened a pull request that should resolve this, #259. Haven't seen any response on it yet, but feel free to try the branch.

**Comment by mathiaswillburger** [comment]
The following solution works for me on Windows with `eslint@7.6.0`, no downgrade required:

Either use `npx` to execute the command directly in your project path and it will recognize `$PWD`, e.g. : `npx prettier-eslint --write $PWD/src/**/*.{js,jsx}` or if you want to have it as script in the `package.json` file (which actually will not recognize `$PWD`)  you'll have to use ` %INIT_CWD%` instead like this:

```
"scripts": {
    "format": "prettier-eslint --write %INIT_CWD%/src/**/*.{js,jsx}",
  },
```

**Comment by kylemh** [comment]
Hey, shut up you. I haven't gotten to it yet.

**Comment by JounQin** [comment]
https://antfu.me/posts/why-reproductions-are-required

Feel free to create a new issue with minimal but runnable reproduction.

**Comment by kylemh** [comment]
FWIW: #259 was merged so this issue should be resolved.
